server/mediaserver.go

```diff
@@ -195,6 +197,20 @@ func (s *LivepeerServer) startBroadcast(cpl core.PlaylistManager) (*BroadcastSes
 	}
 	tinfo := tinfos[0]
 
+	var sessionID string
+
+	if s.LivepeerNode.Sender != nil {
+		protoParams := tinfo.TicketParams
```

Comment:
what's the meaning behind the name `protoParams`? why not stick with `ticketParams`?

Reply:
Oh, I see that `proto` is referring to protobuf. What's the value in defining this variable versus just using `tinfo.TicketParams` ?

Reply:
I defined the additional variable to avoid having multiple instances of `tinfo.TicketParams.<FieldName>` when converting into a `pm.Ticket` struct. However, I'm thinking of just using a helper here:

```
params := net.FromProtoTicketParams(tinfo.TicketParams)
```

WDYT?

Reply:
The helper is fine, but I wouldn't put it in the `net` package since that's become the de facto place for shared type/interface declarations to avoid import cycles.

My feeling is it's best to define methods closest to where they're used, so putting the helper in `rpc.go` (or  somewhere in `server`) is probably better.